app/src/main/java/edu/calvin/cs262/teamc/ProfileActivity.java

```diff
@@ -99,23 +103,23 @@ public void onResponse(String response) {
                     {
 
                     String encodedString = object.getString("profilePhoto");
-                    encodedString = encodedString.replace("data:image/jpeg;base64,","");
+                    encodedString = encodedString
+                            .replace("data:image/jpeg;base64,","");
                     byte[] imageBytes = Base64.decode(encodedString.getBytes(), 0);
-                     currentDefault = BitmapFactory.decodeByteArray(imageBytes,0,imageBytes.length);
+                     currentDefault = BitmapFactory.decodeByteArray(
+                             imageBytes,0,imageBytes.length);
                     image.setImageBitmap(currentDefault);
 
```

Comment:
Might as well delete this empty line too

Reply:
Deleted